packages/ListBox/src/components/Content/Content.styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const ContentStyled = styled.div`
+  :focus {
+    outline: 2px ${tokens.color.blackLighten30} dotted;
```

Comment:
@jayde-acl Changed the dotted line colour to be as you requested

Reply:
Coming soon:
https://github.com/acl-services/paprika/pull/372/files#diff-35881852d236126a5c4a48174b177f8aR88

Reply:
Dotted line will look slightly different using that mixin/helper but we can switch it over when its available. 
![image](https://user-images.githubusercontent.com/4040102/71743439-66160280-2e19-11ea-9438-9612b84291c9.png)